### ISSUE TITLE
Suggestion - Craftable Jewelry 

### ISSUE THREAD

**Issue by spitslave** [InZidiuZ/op-framework-issue-tracker#3612]
### Assurance

I have read the `README.md`

### Summary

So with this suggestion there would also need to be an added Jewelry business, the people who work here will need to receive matts and money from the people who want to purchase the jewelry. Matts can include the things found during mining, as well as adding silver and gold to be able to make the chains. 
That way people don't just sell gems for money and they can put it towards 

So with gem mining and the increase of money being an issue within legacy RP, I think it would be a cool thing to add something expensive for people to buy, and what better option than Jewelry. 
In the mines there are several things people can obtain; Emerald, Sapphire, Diamond, Opal, Tungsten, Morganite, Ruby, Onyx
Check example below
![image](https://github.com/user-attachments/assets/0301d577-fedd-48b8-a8a9-c16ceec31c2f)

We can have several different recipes to make certain types of Jewelry ranging from price and stone used. 
I think that each of these jewelry should be made with a decent amount of stone so that they're not using one of each and the economy wont be fucked by the people who actually make the jewelry. 

Example of Recipes and what can be made; 
Rings, Necklace, Bracelets, Ear Rings 
Necklace Chain -  7 Silver or Gold Nuggets (Can be wearable without the gems as well)
Ring Band - 2 silver or gold nuggets (Can be wearable without the gems as well)
Earrings -  1 silver or gold nuggets 
Bracelets -4 silver or gold nuggets
W/ the addition of gems to make them customizable 

Ruby Necklace  - Chain recipe + 9 processed ruby
Ruby Ring - Ring Recipe + 3 Processed Ruby
Ruby Earring  - Earring recipe + 2 Processed Ruby
Ruby Bracelet - Bracelet Recipe + 7 Processed Ruby

It would be the same recipe for all other types of necklaces, rings, earrings and bracelets 

Could be changed to make more balanced these are just ideas. 

The cost for jeweler to make the jewelry should be quite high if they are going to charge quite high that way there's balance in how much they make. 

### Reason

I feel like people are rich not because they chain rob or grind  but because there is not enough things to buy within the city. I feel like this would add great RP and also something cool for people to spend their money on if they want jewelry made.

**Comment by coalaura** [comment]
Wearable jewelry was suggested previously and is not really possible to do. Jewelry in general sounds cool tho, maybe more crafting spots for the different jewelry

**Comment by spitslave** [comment]
How do the pets work? I’m not really a dev and don’t understand this stuff
But I know if a pet is in the inventory of someone it shows up on their shoulder. And I wanted this more so for people to actually have to buy jewerly and not something they can craft again themselves , kinda add more stuff for people to buy so maybe under a business
Engravements and stuff too

**Comment by coalaura** [comment]
> How do the pets work? I’m not really a dev and don’t understand this stuff. But I know if a pet is in the inventory of someone it shows up on their shoulder. And I wanted this more so for people to actually have to buy jewerly and not something they can craft again themselves , kinda add more stuff for people to buy so maybe under a business

pets work as they are simple and just need to be attached to one point. jewelry would require actual clothing, etc. generally a lot more difficult so i'd say no to the wearable part. I like the rest of the suggestion tho

**Comment by spitslave** [comment]
Yeah I'd definitely want it to be a business and somebody to own so that people can "buy jewelry" rather than crafting them, just think people need something to buy. If they were wearable I feel like they'd be more desirable but I understand that that can't work.

**Comment by coalaura** [comment]
The problem is that if you make it so they have to buy the jewelry then it makes no sense for it to be a player owned business. If they get the materials delivered to them then they should craft it not buy it. I get the idea is to remove more money from the economy but it'd be weird to have to buy the jewelry for a lot of money if you have the materials for it right there. If i implemented this i'd make it a crafting spot like how you can craft the rings. That also adds a lot more sentimental value to the items vs them just being bought off a local vendor

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
